llvm/lib/Bitcode/Reader/BitcodeReader.cpp

```diff
@@ -1117,6 +1117,21 @@ static GlobalVarSummary::GVarFlags getDecodedGVarFlags(uint64_t RawFlags) {
       (GlobalObject::VCallVisibility)(RawFlags >> 3));
 }
 
+static void getDecodedHotnessCallEdgeInfo(uint64_t RawFlags,
```

Comment:
nit: Prefer to return the Hotness and TailCall attribute as a tiny struct or a pair instead of using references.

Reply:
done